integration/lsp/jasmine.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I'm keeping this config file around to make it easier to turn off randomization when debugging.